dev/com.ibm.websphere.appserver.features/visibility/public/microProfile-2.2/resources/l10n/com.ibm.websphere.appserver.microProfile-2.2_cs.properties

```diff
@@ -0,0 +1,17 @@
+###############################################################################
+# Copyright (c) 2016 IBM Corporation and others.
```

Comment:
Defintiely wrong date

Reply:
The information in this file is substantively the same as this file:

dev/com.ibm.websphere.appserver.features/visibility/public/microProfile-2.2/resources/l10n/com.ibm.websphere.appserver.microProfile-2.2_cs.properties

which has the 2016 date. Given there is only 1 character difference I don't think that constitutes a substantial enough change for a different copyright year.